### PR TITLE
blog: rust client 0.1 release

### PR DESCRIPTION
**Pull request by nrc**
Adds a blog post to announce the 0.1 release of the Rust client

PTAL @dcalvin @WT-Liu

### CODE REVIEW

File: content/blog/client-rust-0.1.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+---
+title: TiKV Rust Client - 0.1 release
+date: 2021-05-14
+author: Nick Camerin
+tags: ['Release', 'Announcement']
+---
+
+We're pleased to announce the 0.1 release of the [TiKV Rust client](https://crates.io/crates/tikv-client).
+
+[TiKV](https://tikv.org/) is a distributed key-value store. TiKV is powerful, mature, and widely used as part of [TiDB](https://pingcap.com/products/tidb/) (a 'NewSQL' database). It is [open source](https://github.com/tikv/tikv) and written in Rust. However, up until now it has been very difficult to use as a standalone store in a Rust program. With the Rust client, that has changed, it is now easy to access reliable, persistent storage from Rust.
```
Comment (WT-Liu) [inline content/blog/client-rust-0.1.md:10]:
```suggestion
[TiKV](https://tikv.org/) is a distributed key-value store. TiKV is powerful, mature, and widely used as part of [TiDB](https://pingcap.com/products/tidb/) (a 'NewSQL' database). It is [open source](https://github.com/tikv/tikv) and written in Rust. However, up until now it has been very difficult to use as a standalone store in a Rust program. With the Rust client, that has changed; it is now easy to access reliable, persistent storage from Rust.
```

**Review by WT-Liu — APPROVED** [review]
Rest LGTM. Thanks!

**Comment by nrc** [comment]
@WT-Liu comments addressed, thanks!

File: content/blog/client-rust-0.1.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+tiup playground nightly --kv-only
+```
+
+Then you just need to `cargo run` to run your program using the client.
+
+There is more info in our [docs](https://docs.rs/tikv-client). If you want more help, the best place to go is the #client-rust channel in the [TiKV Slack](https://tikv.org/chat).
+
+## Ongoing development
+
+Today marks the 0.1 release of TiKV's Rust client. As the version number suggests, there is lots more [work to do](https://github.com/tikv/client-rust/pull/290/files) (and the current release is not suitable for use in production). The client is not yet feature complete; although there is good support for all the fundamentals, TiKV is a mature product and has lots of options, features, and customisations, so there is lots still to add to the client. There is also ongoing work to improve reliability, ergonomics, and performance.
```
Comment (dcalvin) [inline content/blog/client-rust-0.1.md:105]:
```suggestion
Today marks the 0.1 release of TiKV's Rust client. As the version number suggests, there is lots more [work to do](https://github.com/tikv/client-rust/pull/290/files), and the current release is not suitable for use in production). There is good support for all the fundamentals. However, the client is not yet feature-complete, with lots of options, features, and customizations to add to the client. There is also ongoing work to improve reliability, ergonomics, and performance.
```

File: content/blog/client-rust-0.1.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+We're pleased to announce the 0.1 release of the [TiKV Rust client](https://crates.io/crates/tikv-client).
+
+[TiKV](https://tikv.org/) is a distributed key-value store. TiKV is powerful, mature, and widely used as part of [TiDB](https://pingcap.com/products/tidb/) (a 'NewSQL' database). It is [open source](https://github.com/tikv/tikv) and written in Rust. However, up until now it has been very difficult to use as a standalone store in a Rust program. With the Rust client, that has changed; it is now easy to access reliable, persistent storage from Rust.
+
+## What is TiKV?
+
+TiKV is a key-value store, like a hashmap but reliably persistent and scalable to terrabytes of data. It runs across multiple nodes for reliability, availability, and scalability - even if you lose some nodes, your data will never be lost and will still be readable and writable; as your data grows, you can simply add more nodes.
+
+TiKV supports transactions, like traditional SQL databases, and is ACID-compliant. I.e., transactions are atomic, isolated (with snapshot isolation), and durable.
```
Comment (dcalvin) [inline content/blog/client-rust-0.1.md:16]:
```suggestion
TiKV supports transactions, like traditional SQL databases, and is ACID-compliant, i.e., transactions are atomic, isolated (with snapshot isolation), and durable.
```

**Comment by dcalvin** [comment]
The rest LGTM @nrc

**Comment by nrc** [comment]
@dcalvin comments addressed, thanks!

**Review by dcalvin — APPROVED** [review]
LGTM. Thanks.

**Pull request merged**